tools/changelog/src/update-latest-versions.js

```diff
@@ -0,0 +1,79 @@
+import { remark } from 'remark';
+import stringify from 'remark-stringify';
+import remarkParse from 'remark-parse';
+import fs from 'fs';
+import path from 'path';
+import { format } from 'date-fns';
+import { writeFile } from 'fs/promises';
+import getVersionHistory from './update-version-histories.js';
+
+const packagesDir = '../../packages';
+const versionOnlyRegex = /^\d+\.\d+\.\d+$/;
+
+function getAdaptorsFromDir() {
```

Comment:
This function is used in both script - please move it out into a common utils.js file